hack/generate-operator-bundle-contents.py

```diff
@@ -1,35 +1,47 @@
 #!/usr/bin/env python3
 
+# vim: ts=4:sw=4:cc=99
+
 import datetime
 import os
 import sys
 import yaml
 import pathlib
 import argparse
 
-parser = argparse.ArgumentParser()
-parser.add_argument("-r", "--replaces", help="Replaces version", type=str)
-parser.add_argument('-s','--skip', action='append',
+parser = argparse.ArgumentParser(add_help=False)
+required = parser.add_argument_group('required arguments')
+
+required.add_argument('-n', '--name', help='operator name', type=str, required=True)
+required.add_argument('-v', '--version', help='operator version', type=str, required=True)
+required.add_argument('-i', '--image', help='operator image', type=str, required=True)
+required.add_argument('-t', '--image-tag', help='operator image tag', type=str, required=True)
+required.add_argument('-o', '--output-dir', help='output directory', type=str, required=True)
+
+optional = parser.add_argument_group('optional arguments')
+optional.add_argument( '-h', '--help', action='help', default=argparse.SUPPRESS,
+    help='show this help message and exit')
+optional.add_argument('-r', '--replaces', help='Replaces version', type=str)
+optional.add_argument('-s','--skip', action='append',
         help='Skips version (can be specified multiple times)')
+
 args = parser.parse_args()
 
-root = pathlib.Path(__file__).parent.absolute() / '../..'
+root = pathlib.Path(__file__).parent.absolute() / '..'
 manifest_dir = root / 'deploy/openshift'
 csv_template_dir = root / 'deploy/bundle/template'
 
 with open(csv_template_dir / 'deploymentvalidationoperator.clusterserviceversion.yaml',
```

Comment:
Should this use some form of `args.name`?

(Perhaps it would be simplest to expect the name of the template to include the hyphens. So in this case `deployment-validation-operator.clusterserviceversion.yaml`.)

Reply:
totally I will change it

Reply:
fixed in 9f2a90b